tmt/steps/report/display.py

```diff
@@ -80,11 +83,19 @@ def _display_log_content(log: Path, shift: int) -> None:
         def display_outcome(result: BaseResult) -> None:
             """ Display a single result outcome """
 
-            if isinstance(result, CheckResult):
+            if isinstance(result, SubCheckResult):
+                self.verbose(
+                    f'{result.show()} ({result.event.value} check)',
+                    shift=SUBRESULT_CHECK_SHIFT)
+
+            elif isinstance(result, CheckResult):
                 self.verbose(
                     f'{result.show()} ({result.event.value} check)',
                     shift=CHECK_SHIFT)
 
+            elif isinstance(result, SubResult):
+                self.verbose(f'{result.show()}', shift=SUBRESULT_SHIFT)
```

Comment:
```suggestion
                self.verbose(result.show(), shift=SUBRESULT_SHIFT)
```

Not necessary to use the f-string here?

Reply:
Thanks, resolved.